modules/nf-core/biscuit/biscuitblaster/main.nf

```diff
@@ -27,6 +27,7 @@ process BISCUIT_BLASTER {
     def biscuit_cpus = (int) Math.max(Math.floor(task.cpus*0.95),1)
     def samtools_cpus = task.cpus-biscuit_cpus
     """
+    set -o pipefail
```

Comment:
nf-core pipelines already have this, cf https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L175